data/image_folder.py

```diff
@@ -21,7 +21,7 @@ def is_image_file(filename):
     return any(filename.endswith(extension) for extension in IMG_EXTENSIONS)
 
 
-def make_dataset(dir, max_dataset_size=float("inf")):
+def make_dataset(dir, batch_size, max_dataset_size=float("inf"), val_eval=False):
```

Comment:
This breaks other files that make a call to `make_dataset()` without passing batch_size. Try passing batch_size as an optional param after max_dataset_size.

Reply:
Also params batch_size and val_eval are not used anywhere here